responses/matchers.py

```diff
@@ -53,7 +53,9 @@ def list_to_str(input_list: List[str]) -> str:
     return key_val_str
 
 
-def urlencoded_params_matcher(params: Optional[Dict[str, str]]) -> Callable[..., Any]:
+def urlencoded_params_matcher(
+    params: Optional[Dict[str, str]], allow_blank: bool = False
```

Comment:
```suggestion
    params: Optional[Dict[str, str]], *, allow_blank: bool = False
```

Need to force only kwargs

Reply:
Fixed in [c50cdf7](https://github.com/getsentry/responses/pull/534/commits/c50cdf7e25597e538b2b887df38ab8eef2439dfc)